src/tasks/platforms/minikube.ts

```diff
@@ -131,6 +131,55 @@ export class MinikubeTasks {
     ], { renderer: flags['listr-renderer'] as any })
   }
 
+  configureApiServer(flags: any): ReadonlyArray<Listr.ListrTask> {
```

Comment:
Please give more specific name for the function, for example `configureApiServerForDex`.